packages/sitebuilder/src/contexts/GoLiveProvider.tsx

```diff
@@ -94,46 +96,35 @@ const GoLiveProvider = ({ children }: { children: React.ReactNode }) => {
 	}, [retryVerfication]);
```

Comment:
In `retryVerfication` the word "verification" is spelled wrong. It should be `retryVerification`.